lang/syn/src/idl/defined.rs

```diff
@@ -496,79 +496,90 @@ pub fn gen_idl_type(
                 use crate::parser::context::CrateContext;
                 use quote::ToTokens;
 
-                let source_path = proc_macro2::Span::call_site().source_file().path();
-                if let Ok(Ok(ctx)) = find_path("lib.rs", &source_path).map(CrateContext::parse) {
-                    let name = path.path.segments.last().unwrap().ident.to_string();
-                    let alias = ctx.type_aliases().find(|ty| ty.ident == name);
-                    if let Some(alias) = alias {
-                        if let Some(segment) = path.path.segments.last() {
-                            if let syn::PathArguments::AngleBracketed(args) = &segment.arguments {
-                                let inners = args
-                                    .args
-                                    .iter()
-                                    .map(|arg| match arg {
-                                        syn::GenericArgument::Type(ty) => match ty {
-                                            syn::Type::Path(inner_ty) => {
-                                                inner_ty.path.to_token_stream().to_string()
+                if let Some(source_path) = proc_macro2::Span::call_site().local_file() {
+                    if let Ok(Ok(ctx)) = find_path("lib.rs", &source_path).map(CrateContext::parse)
```

Comment:
Nit: The new indentation makes the changes seem bigger than they actually are (+82 −70). It would be great if you could rewrite these lines to use only one indentation.